packages/control/chargepoint/chargepoint.py

```diff
@@ -425,8 +427,8 @@ def check_phase_switch_completed(self):
                     self.data.control_parameter.state = ChargepointState.PHASE_SWITCH_AWAITED
                     if self._is_phase_switch_required() is False:
                         self.data.control_parameter.state = ChargepointState.CHARGING_ALLOWED
-        except Exception:
-            log.exception("Fehler in der Ladepunkt-Klasse von "+str(self.num))
+        except Exception as e:
+            log.exception("Fehler in der Ladepunkt-Klasse von "+str(self.num) + ", error=" + str(e))
```

Comment:
```suggestion
        except Exception:
            log.exception("Fehler in der Ladepunkt-Klasse von "+str(self.num))
```
Die exception-Klasse des Logging-Moduls loggt automatisch den Traceback. Was ist der Untereschied zu Deiner Änderung?